test/unit/pages/patientnew.test.js

```diff
@@ -0,0 +1,92 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var expect = chai.expect;
+var rewire = require('rewire');
+var rewireModule = require('../../utils/rewireModule');
+
+/**
+ * Need to set window.config for config module
+ */
+window.config = {};
+
+describe('PatientNew', function () {
+  var PatientNew = rewire('../../../app/pages/patientnew');
+
+  it('should be exposed as a module and be of type function', function() {
+    expect(PatientNew).to.be.a('function');
+  });
+
+  describe('render', function() {
+    it('should warn when required props are not present', function() {
+      console.warn = sinon.spy();
+      var elem = TestUtils.renderIntoDocument(<PatientNew/>);
+      expect(elem).to.be.ok;
+      expect(console.warn.callCount).to.equal(2);
+      expect(console.warn.calledWith('Warning: Required prop `onSubmit` was not specified in `PatientNew`.')).to.equal(true);
+      expect(console.warn.calledWith('Warning: Required prop `trackMetric` was not specified in `PatientNew`.')).to.equal(true);
+    });
+  });
+
+  describe('getInitialState', function() {
+    it('should be in this expected format', function() {
```

Comment:
I'm not a huge fan of this test description, but then, it is a difficult one to describe...